.github/ETHICAL_USE.md

```
@@ -3,18 +3,20 @@
 
 This project is released under the MIND-UCAL v1.0 License. However, we respectfully request that users and organizations honor the spirit in which Gitmind was created.
 
-### Gitmind Is For:
+### Gitmind Is For
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)